src/state.sh

```diff
@@ -8,6 +8,8 @@ _ASSERTIONS_PASSED=0
 _ASSERTIONS_FAILED=0
 _ASSERTIONS_SKIPPED=0
 _ASSERTIONS_INCOMPLETE=0
+_DUPLICATED_FUNCTION_NAMES=""
+_FILE_WITH_DUPLICATED_FUNCTION_NAMES=""
```

Comment:
I am not sure if I like this to keep it globals, but I guess it would be OK as an MVP idea, and we could refactor it in a follow up if we see the opportunity.